content/community-targets/NESHTEC-NeshNode.md

```diff
@@ -0,0 +1,96 @@
+
+## How to get one?
+Just check out our shop, we'll ship the board from Germany:
+https://neshtec.eu/products/neshnode-controllerplatine
+
+(We are working on a English translation, sorry about that)
+
+## Hardware
+
+NeshNode was designed to be used as flexible as possible in professional projects, without having to develop a new board every time. The board is powerful enough for extensive cryptographic tasks, cloud connectivity (like Microsoft Azure IoT Hub) and offers a lot of interfaces.
+
+### STM32F765VGT6 microcontroller
+
+The board currently uses an ARM Cortex-M7 RISC core based STM32F765VGT6 MCU in a 100 pin package which operates at up to 216 MHz, 2 MB of persistent flash, 512KB SRAM, features a dedicated double precision FPU and also implements a full set of DSP instructions. To increase application security, it also features a MPU (memory protection unit).
+
+See here for more details: https://www.st.com/en/microcontrollers-microprocessors/stm32f765vg.html
+
+### Ethernet
+
+Ethernet is supplied by a Microchip LAN8720A, which is a high performance 10BASE-T/100BASE-TX transceiver delivering up to 100 MBit/s, working on a large temperature range: https://www.microchip.com/en-us/product/lan8720a
+
+### MikroE® Click™ / MikroBus™
+
+MikroE® has designed a compact interface for which, as of today, there are over 1,000 different so-called Click™ boards of very high quality - giving you the flexibility to easily add whatever kind of additional functionality you might need.
+
+See below ([Pinout](#pinout)) for a sample of MikroE®s MikroBus™ layout.
+
+Check out MikroE®'s Click™ board store for some awesome addon boards: https://www.mikroe.com/click
+
+More information about the Click™ standard can be found on the MikroE®s website: https://www.mikroe.com
+
+### USB-C connector
+This can be used for powering the board, charging the battery (if installed) at the same time. Or you may use the USB-C connector to upload and debug nanoFramework programs from within Visual Studio (or any other IDE supported by nanoFramework) once you flashed the nanoFramework bootloader.
+
+### Programming and debug interface
+A fully connected programming and debug interface, which comes without headers by default, can be used by native STMicroelectronics tools for the STM32 line of MCUs. For example, you may use STLink for flashing the nanoFramework bootloader, to set fuses and so on.
+
+### External I2C connector
+There's a dedicated connector sporting 3V3, GND, SCL, SDA and RST on the rear side of the board. This is supposed to be used for I2C, but you may also reconfigure the pins to act as GPIO.
+
+### Battery support
+The power management controller ([see below, LTC3586](#ltc3586-power-management)) supports LiIon- and LiPo-Batteries. You may use one, but you don't have to. The board automatically detects any installed battery and starts using and charging it, if a sufficent external power supply is available.
```

Comment:
Is there any binding that does support this in the IoT device repository? If not, possible to add a simple one? See https://github.com/nanoframework/nanoFramework.IoT.Device